packages/types/src/Metadata/v3/Storage.ts

```diff
@@ -25,39 +27,39 @@ export {
 export class DoubleMapType extends Struct {
   constructor (value?: any) {
     super({
-      key1: Text,
-      key2: Text,
-      value: Text,
-      key2Hasher: Text
+      key1: Type,
+      key2: Type,
+      value: Type,
```

Comment:
The above 3 are all Text, revert

Reply:
those 3 need to be Type otherwise it won’t do the type preprocess step and causing issues. e.g. T::Balance need the type process step to convert to Balance then can be found in the type registry

Reply:
Ah yeah, you're totally right